build/elasticsearch/bin/es-docker

```diff
@@ -22,4 +22,6 @@ do
     fi
 done < <(env)
 
+echo "-Des.cgroups.hierarchy.override=/" >> config/jvm.options
```

Comment:
Correct me if I'm wrong please, but I don't think we can do it in the Dockerfile since that bakes the value of ES_JAVA_OPTS into the image but users can override this externally when running the image.

I will push a comment explaining the situation later this morning.